index.html

```diff
@@ -17,20 +17,128 @@
+                        <li class="BulleDesign">Back End Dev</li>
+                        <li class="BulleDesign">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                        <ul class="design infolist">
+                            <li class="liststyle design span">Html</li> 
+                            <li class="liststyle design span">Css</li> 
+                            <li class="liststyle design span">Javascript</li> 
+                        </ul>
+                        <button class="btn">See Project</button>
+                </div>
+            </div>
+            <div  class="card">
+                <img class="gridimg"  src="images/Snapshoot3Portfolio.png"  alt="ThirdProjectScreenShot">
+                <div class="TextArea">
+                    <h3>Tonic</h3>
+                    <ul class="infolist">
+                        <li class="design font ">CANOPY</li>
+                        <li class="BulleDesign">Back End Dev</li>
+                        <li class="BulleDesign">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                        <ul class="design infolist">
+                            <li class="liststyle design span">Html</li> 
+                            <li class="liststyle design span">Css</li> 
+                             <li class="liststyle design span">Javascript</li> 
+                        </ul>
+                        <button class="btn">See Project</button>
+                </div>
+            </div>
+            <div  class="card">
+                <img class="gridimg"  src="images/Snapshoot5Portfolio.png"  alt="FourthProjectScreenshoot">
+                <div class="TextArea">
+                    <h3>Multi-Post
+                      <br>Stories</h3>
+                    <ul class="infolist">
+                        <li class="design font ">CANOPY</li>
+                        <li class="BulleDesign">Back End Dev</li>
+                        <li class="BulleDesign">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                        <ul class="design infolist">
+                            <li class="liststyle design span">Html</li> 
+                            <li class="liststyle design span">Css</li> 
+                             <li class="liststyle design span">Javascript</li> 
+                        </ul>
+                        <button class="btn">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section id="About" class="headlineSection">               
+            <h1>About
+                <br> Myself</h1>
```

Comment:
- [ ] Kindly change this `h1` tag into any other hx tag of your choice. It is a best practice to reserve the `h1` tag to be used as a title to the page and only once